src/components/UserDetail/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
```

Comment:
i would move all these files in their own subdirectory under src/components/profile or something if you haven't already. i would guess if you guys have been putting everything under src/components its really hard to figure out whats going on in that directory because theres like 20 components?

Reply:
I agree with that. we follow this method in the board project, but in this project, we put each component in a separate directory. However, I will do it right know.